serialize.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016-2019 Alex Stocks, Xinge Gao
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hessian
+
+type Serializer interface {
```

Comment:
change the file `serialize.go` to and `serializer.go`
- move all decimal serialize code to `serializer.go`
- move the code in `registrar.go` to `serializer.go`

Reply:
Please wait a moment